src/main/java/com/ioextendedgr/web/controller/ConferenceController.java

```diff
@@ -9,6 +9,7 @@
 import com.ioextendedgr.web.viewDto.ConferenceView;
 
 @RestController
+@RequestMapping("/api")
```

Comment:
A strong move here. I was hoping we'd do this.  Also a key change to communicate out to make sure people know the right URL to test this against.